src/v0/destinations/tiktok_ads/transformV2.js

```diff
@@ -392,23 +420,23 @@ const buildBatchResponseForEvent = (inputEvent) => {
   return batchedRequest;
 };
 
-const getEventChunks = (event, trackResponseList, eventsChunk) => {
+const separateTestEvents = (event, processedTestEvents, processedEvents) => {
   // only for already transformed payload
   // eslint-disable-next-line no-param-reassign
   event.message = Array.isArray(event.message) ? event.message : [event.message];
 
   // not performing batching for test events as it is not supported
   if (event.message[0].body.JSON.test_event_code) {
```

Comment:
what does test_event_code mean & how it gets generated ?

Reply:
test_event_code is used to verify the setup before production as per [this](https://business-api.tiktok.com/portal/docs?id=1771101049613313) doc. We are populating it from `properties.testEventCode`